troposphere/static/js/AppRoutes.react.js

```diff
@@ -76,7 +77,8 @@ define(function (require) {
       <Route name="settings" handler={SettingsPage}/>
 
       <Route name="admin" handler={AdminMaster}>
-        <DefaultRoute name="identity-membership-manager" handler={IdentityMembershipMaster}/>
+        <DefaultRoute name="atmosphere-user-manager" path="users" handler={AtmosphereUserMaster}/>
```

Comment:
This is an issue Andy caught with my last PR. DefaultRoute doesn't expect a path prop. It will complain. You should also see that the path isn't added to the url even though the link is active and the handler has been executed. That is the correct behavior.